tools/tsTreeshaker.ts

```diff
@@ -0,0 +1,258 @@
+// code based on ts-transform-paths
```

Comment:
What does this file do?

Why copy code from `ts-transform-paths` instead of using it as plugin?

Reply:
This is not the `ts-transform-paths` plugin, it just based on it.

I made it because there is no tree shaker for typescript build.

Reply:
![image](https://user-images.githubusercontent.com/48761044/190594751-4ad20d06-91d8-47c6-a6a9-bcad59393470.png)
This is a picture of the build pipeline as it is. I made it to help you understand.
tsTreeshaker is in charge of tree shake transform here